src/contexts/AuthContext.tsx

```
@@ -9,6 +9,7 @@ import {
 } from "react";
 import axios from "axios";
 import { useRouter } from "next/navigation";
+import { api } from "@/api/axios"; // 기존 API 인스턴스 사용
```

ESLint

[error] 'api' is defined but never used.

(@typescript-eslint/no-unused-vars)